render-tests/line-opacity/property-function/style.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
haha I was wondering about that haha -- we're setting line-blur instead of line-opacity 🙈. updating in a sec.